src/hooks/useCheckAccess.ts

```diff
@@ -18,10 +18,10 @@ const useCheckAccess = () => {
   const location = useLocation();
   const [errorMessage, setErrorMessage] = useState<string | null>(null);
   const { t } = useTranslation('user');
+  const { clientOffers } = useClientIntegration();
```

Comment:
Should we also add it as a dependency to the callback?

Reply:
Yes, it does! I realized that hooks weren't included by ESLint due to the file extension. I changed that, but that brought some other changes as well 😅